sle15/cpe/sle15-cpe-dictionary.xml

```diff
@@ -67,9 +67,14 @@
             <!-- the check references an OVAL file that contains an inventory definition -->
             <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_systemd_package</check>
       </cpe-item>
-      <cpe-item name="cpe:/a:yum">
-            <title xml:lang="en-us">Package yum is installed</title>
+      <cpe-item name="cpe:/a:zypper">
+            <title xml:lang="en-us">Package zypper is installed</title>
             <!-- the check references an OVAL file that contains an inventory definition -->
-            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_yum_package</check>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_env_has_zypper_package</check>
+      </cpe-item>
+      <cpe-item name="cpe:/a:ssh:secure_shell">
```

Comment:
This shouldn't be needed as the existing ssh checks should handle this.

Reply:
removed in latest commit